npc/jobs/2-1/assassin.txt

```diff
@@ -2064,7 +2057,8 @@ OnTouch_:
 			mes "I believe you stand out amongst them...";
 			next;
 		}
-		if (ASSIN_Q3 == 1 && BaseJob == Job_Thief && ASSIN_Q > 6 && ASSIN_Q < 17) {
```

Comment:
This is a giant OnTouch event, but is the removal of the Job check ok here? Also, the removal of the `ASSIN_Q` part doesn't seem to be anywhere below as well, is that ok?

Reply:
Yeah, both are already checked right at the start in line 1644
`if (ASSIN_Q == 7 && BaseJob == Job_Thief) {`
And there's no way ASSIN_Q gets a value out of those limits during all the code, even if the player relogs or times out, the value gets reseted to 7, so that check was just redundant